### PR TITLE
[wip] Release notes for PrestaShop 8.1

### PR DESCRIPTION
**Pull request by kpodemski**
8.1 is coming!

### CODE REVIEW

File: content/news/releases/prestashop-8-1-0-available.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+A big thank you to the X people who contributed to Y Pull Requests included in this version:
+
+TO DO
+
+PrestaShop is above all a community project: the vast majority of contributors are not directly affiliated with PrestaShop SA.
+
+We would also like to thank the people who contributed aside from code:
+
+- Product management: Fabien Vallon, Mateus Shirlaw, Zoé Finders and Rosa Benouamer 
+- Quality assurance: Aurélien Rita, Faten Mhiri, Florine Hea, Hana Rebai, Hibatallah Aouadni, Ines Sallemi, Khouloud Belguith, Marwa Chelly, Paul-Noël Cholot, Robin Fischer, Sarah Cavrot, Sarah Dib, Serhii Zavadskyi and Sira Diouara.
```
Comment (Robin-Fischer-PS) [inline content/news/releases/prestashop-8-1-0-available.md]:
```suggestion
- Quality assurance: Achraf Kchaou, Anis Zouari, Aurélien Rita, Emna Chouichi, Faten Mhiri, Florine Hea, Hana Rebai, Hejer Elleuch, Hibatallah Aouadni, Ines Sallemi, Khouloud Belguith, Paul-Noël Cholot, Robin Fischer, Sarah Dib and Serhii Zavadskyi.
```

**Review by Robin-Fischer-PS — COMMENTED** [review]
Added the QA members in suggested change, otherwise looks good, thanks :)

File: content/news/releases/prestashop-8-1-0-available.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title:  PrestaShop 8.1 Is Available
+subtitle: Minor version, major improvements!
+date: 2023-06-20
```
Comment (coffrep) [inline content/news/releases/prestashop-8-1-0-available.md]: Date to modify before publishing

File: content/news/releases/prestashop-8-1-0-available.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+- releases
+- "8.1"
+- "8.1.x"
+---
+
+We are pleased to announce that PrestaShop 8.1 is officially available.
+
+![PrestaShop 8.1 is available!](/assets/images/2023/02/banner_8.1_1534x424.png)
+
+PrestaShop 8.1, is a minor upgrade with major improvements. It's focused on making things faster and easier for you, transforming the way you use PrestaShop.
```
Comment (coffrep) [inline content/news/releases/prestashop-8-1-0-available.md]:
```suggestion
PrestaShop 8.1 is a minor upgrade with major improvements. It's focused on making things faster and easier for you, transforming the way you use PrestaShop.
```

File: content/news/releases/prestashop-8-1-0-available.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+The entirely reworked product management page delivers a more efficient and user-friendly experience. This new page make managing product combinations, prices, taxes, stock, and images quicker and easier. Multistore product management also has been drastically improved. Performance-wise, the new product management page is a whole new level, especially for stores with a large number of combinations.
```
Comment (coffrep) [inline content/news/releases/prestashop-8-1-0-available.md]:
```suggestion
The entirely reworked product management page delivers a more efficient and user-friendly experience. This new page makes managing product combinations, prices, taxes, stock, and images quicker and easier. Multistore product management has also been drastically improved. Performance-wise, the new product management page is taken to a new level, especially for stores with a large number of combinations.
```

File: content/news/releases/prestashop-8-1-0-available.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+Additional new features include improved control over product availability labels, enhanced SEO settings for products, and support for multiple format image generation (WebP, AVIF). There are also a few improvements for developers, such as simiplified access to the store's performance profiler.
```
Comment (coffrep) [inline content/news/releases/prestashop-8-1-0-available.md]:
```suggestion
Additional new features include improved control over product availability labels, enhanced SEO settings for products, and support for multiple format image generation (WebP, AVIF). There are also a few improvements for developers, such as simplified access to the store's performance profiler.
```

File: content/news/releases/prestashop-8-1-0-available.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+A big thank you to the X people who contributed to Y Pull Requests included in this version:
+
+TO DO
+
+PrestaShop is above all a community project: the vast majority of contributors are not directly affiliated with PrestaShop SA.
+
+We would also like to thank the people who contributed aside from code:
+
+- Product management: Fabien Vallon, Mateus Shirlaw, Zoé Finders and Rosa Benouamer
```
Comment (coffrep) [inline content/news/releases/prestashop-8-1-0-available.md]:
```suggestion
- Product management: Fabien Vallon, Mateus Shirlaw, Zoé Finders and Rosa Benouamer.
```

**Review by coffrep — DISMISSED** [review]
Hi Krystian, great article for 8.1! I'm requesting changes on this file. It would be great if you could let me re-read the article once every edit has been made (adding contributors, numbers, etc...).

**Pull request merged**